data_steward/validation/metrics/required_labs.py

```diff
@@ -54,6 +67,17 @@ def load_measurement_concept_sets_descendants_table(project_id, dataset_id):
     :return: None
     """
 
+    client = bq.get_client(project_id)
+    table_name = f'{project_id}.{dataset_id}.{MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE}'
+
+    if not bq_utils.table_exists(MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE,
+                                 dataset_id):
+        bq.create_tables(client=client,
+                         project_id=project_id,
+                         fq_table_names=[table_name],
+                         exists_ok=False,
+                         fields=None)
+
```

Comment:
The ticket also mentions copying the vocabulary files if missing. This can be added here, before the query `IDENTIFY_LABS_QUERY` is called.